dask/base.py

```diff
@@ -1225,13 +1225,13 @@ def get_collection_names(collection) -> Set[str]:
     Examples
     --------
     >>> a.__dask_keys__()  # doctest: +SKIP
-    ["foo", "bar"]  # doctest: +SKIP
+    ["foo", "bar"]
```

Comment:
I didn't realize that the `doctest: +SKIP` is only required on the input. This is a nice cleanup!